web/src/ResourceInfo.tsx

```diff
@@ -48,7 +48,10 @@ let PortForwardLabel = styled.span`
 
   ${s.mixinHideOnSmallScreen}
 `
-let PortForwardLink = styled.a``
+
+let PortForwardLink = styled.a`
```

Comment:
This would add extra spacing to the right of every single Port Forward link, when what we really want is to add spacing between links when there is more than one. 

So, I would replace the line you added with this CSS adjacent sibling selector, so you end up with this:

```
let PortForwardLink = styled.a`
  & + &  {
    padding-left: ${s.SizeUnit(0.25)};
    border-left: 1px dotted ${s.Color.grayLight};
    margin-left: ${s.SizeUnit(0.25)};
  }
`
```

Reply:
Cool! Didn't know you could put selectors right here. Thanks!

Reply:
Looks great!

<img width="711" alt="Screen Shot 2020-02-24 at 2 24 45 PM" src="https://user-images.githubusercontent.com/10860550/75183926-7cea1e80-5711-11ea-8a6c-a4940f4e6384.png">